finagle-core/src/main/scala/com/twitter/finagle/tracing/TraceInitializerFilter.scala

```diff
@@ -243,11 +243,24 @@ private[finagle] object WireTracingFilter {
 
   def module[Req, Rep]: Stackable[ServiceFactory[Req, Rep]] =
```

Comment:
Perhaps renaming this `module` to `clientModule` to make thing symmetric? Should be safe to do since it's package-private for `[finagle]`.